docs/container.md

```diff
@@ -0,0 +1,69 @@
+<!--
+SPDX-FileCopyrightText: Red Hat, Inc.
+SPDX-License-Identifier: GPL-2.0-or-later
+-->
+
+# Containerized imageio server
+
+## Overview
+
+ovirt-imageio provides an image for running the ovirt-imageio-daemon
+in a container.
```

Comment:
Maybe:

    You can use ovirt-imageio to transfer images without oVirt using the
    ovirt-imageio container image.